### PR TITLE
Cm 12.1

### PR DESCRIPTION
**Pull request by yupi731**

### CODE REVIEW

**Comment by yupi731** [comment]
Thanks

**Comment by yupi731** [comment]
Thanks all

**Comment by luk1337** [comment]
Could you fucking stop spamming my github feed?

File: arch/arm/configs/cyanogenmod_i9082_defconfig [hunk 1]
```diff
@@ -46,7 +46,7 @@ CONFIG_KERNEL_GZIP=y
 # CONFIG_KERNEL_LZMA is not set
 # CONFIG_KERNEL_LZO is not set
 CONFIG_DEFAULT_HOSTNAME="(none)"
-# CONFIG_SWAP is not set
```
Comment (yupi731) [inline arch/arm/configs/cyanogenmod_i9082_defconfig]: +CONFIG_SWAP=y

Comment (yupi731) [inline arch/arm/configs/cyanogenmod_i9082_defconfig]: +CONFIG_SWAP=y

File: arch/arm/configs/cyanogenmod_i9082_defconfig [hunk 2]
```diff
@@ -46,7 +46,7 @@ CONFIG_KERNEL_GZIP=y
 # CONFIG_KERNEL_LZMA is not set
 # CONFIG_KERNEL_LZO is not set
 CONFIG_DEFAULT_HOSTNAME="(none)"
```
Comment (yupi731) [inline arch/arm/configs/cyanogenmod_i9082_defconfig]: Thanks

**Pull request closed without merging**